pgml-extension/src/bindings/transformers/mod.rs

```diff
@@ -69,6 +83,7 @@ pub fn embed(
     inputs: Vec<&str>,
     kwargs: &serde_json::Value,
 ) -> Result<Vec<Vec<f32>>> {
+    set_env();
```

Comment:
What's the latency/lock contention cost of accessing the GUC on every call of embed?

Reply:
The cost is acquiring the mutex lock, building a treemap in `gen_hf_env_map`, and comparing two treemaps. It seems a little inefficient.